### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by serentinos**
[DEMO LINK](https://serentinos.github.io/react_static-list-of-posts/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,109 +1,25 @@
...
-
-          <p>
-            {' Posted by  '}
-
-            <a className="UserInfo" href="mailto:Julianne.OConner@kory.org">
-              Patricia Lebsack
-            </a>
-          </p>
-        </div>
+const fullPostData = prepareApiData({
```
Comment (Ihor-Karpyn) [inline src/App.tsx:12]: Let's rename this const to `posts`

File: src/components/CommentInfo/CommentInfo.tsx [hunk 2]
```diff
@@ -1,5 +1,34 @@
-import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+interface CommentInfoProps {
```
Comment (Ihor-Karpyn) [inline src/components/CommentInfo/CommentInfo.tsx:3]: just Props

File: src/components/CommentList/CommentList.tsx [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+interface CommentListProps {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<CommentListProps> = ({ comments }) => {
+  const isHaveComments = comments.length > 0;
+  const commentsAmmount = comments.length;
```
Comment (Ihor-Karpyn) [inline src/components/CommentList/CommentList.tsx:10]:
```suggestion
  const commentsAmmount = comments.length;
  const hasComments = commentsAmmount > 0;
```

File: src/components/PostInfo/PostInfo.tsx [hunk 4]
```diff
@@ -1,5 +1,32 @@
-import React from 'react';
+import { FullPostData } from '../../types/FullPostData';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+interface PostInfoProps {
```
Comment (Ihor-Karpyn) [inline src/components/PostInfo/PostInfo.tsx:5]: just Props

File: src/components/PostList/PostList.tsx [hunk 5]
```diff
@@ -1,5 +1,16 @@
-import React from 'react';
+import { FullPostData } from '../../types/FullPostData';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
-);
+interface PostListProps {
```
Comment (Ihor-Karpyn) [inline src/components/PostList/PostList.tsx:4]: Props

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]
GJ, almost done

File: src/components/CommentList/CommentList.tsx (hunk 3 above)
Comment (dinaminahi) [reply src/components/CommentList/CommentList.tsx:10]: Please fix @Ihor-Karpyn comment and we are done

**Review by dinaminahi — CHANGES_REQUESTED** [review]

**Comment by serentinos** [comment]
@dinaminahi Sory, didn't noticed that

**Review by dinaminahi — APPROVED** [review]
Well done 👍

**Pull request closed without merging**